### PR TITLE
Added a transactions button for the manager

### PR DESCRIPTION
**Pull request by calvinbrown085**

### CODE REVIEW

File: templates/Managerpage.html [hunk 1]
```diff
@@ -54,6 +54,7 @@
 </tr>
 </table>
 <p><input type="submit" name="Update" value="Update">
+<a href="/transactions"><button type="button">Transactions</button></a><br><br>
```
Comment (ethanphunter) [inline templates/Managerpage.html:57]: How does this work? does it look ok? Would it look better with the `<a>` tag to be inside the `<button>` tag? Or would it look the same?

**Comment by ethanphunter** [comment]
:+1:

**Comment by calvinbrown085** [comment]
image

**Comment by calvinbrown085** [comment]
@ethanphunter This is what the button looks like if thats what you mean

**Comment by calvinbrown085** [comment]
I need chad to adjust it, but it looks fine

**Comment by ethanphunter** [comment]
Looks fine to me

**Comment by calvinbrown085** [comment]
image

**Comment by gilmerc** [comment]
same

**Comment by ethanphunter** [comment]
Merrrrrrrrrrggggginnnnngggggg

**Pull request merged**